frontend/components/layout/StagingBanner.stories.js

```diff
@@ -7,5 +7,5 @@ const common = {
 
 storiesOf('Layout/StagingBanner', module).add('Default', () => ({
     ...common,
-    template: `<StagingBanner />`,
+    template: `<StagingBanner url="https://auctions.makerdao.network/" />`,
```

Comment:
```suggestion
    template: `<StagingBanner url="https://example.com/" />`,
```

This will make it obvious that it's configurable :upside_down_face: